syft/frameworks/torch/mpc/aby3/aby3_helper.py

```diff
@@ -0,0 +1,48 @@
+from syft.frameworks.torch.mpc.falcon.falcon_helper import FalconHelper
+
+
+class ABY3Helper:
+    def bit_inject(rst_binary):
```

Comment:
Can you add to the signature an argument `ring_size` defaulting to 2 ** 32 that you propagate to `convert_b2a` ?